### DISCUSSION TITLE
About coco in Project-MONAI/tutorials/tree/main/detection

### DISCUSSION THREAD

**Discussion by Dana136484** [Project-MONAI/tutorials#1624] [Q&A]
When calculating the validation metrics after several rounds of training, coco is calculated for all classes, which results in the optimal metric being negative. But I used my own dataset for training, and the category labels of all bounding boxes in a training set only contain some classes, so only the category labels of the bounding boxes in ground truth need to be calculated.How should I modify it?Thanks!
 all classes:
![所有类别标签](https://github.com/Project-MONAI/tutorials/assets/126674286/d7307861-1b01-44e2-82aa-8114ff7471b5)
![对所有类别标签进行循环](https://github.com/Project-MONAI/tutorials/assets/126674286/13e8275b-9256-4053-8851-a66e4aa06cc7)
The bounding box category labels for my own data:
![真值标签](https://github.com/Project-MONAI/tutorials/assets/126674286/70a6da53-242c-46a6-90d6-478e8ba852c3)
